modules/monitoring/application/views/scripts/partials/downtime/downtime-header.phtml

```diff
@@ -10,21 +10,41 @@
 <td>
   <div class="comment-author">
     <?php if ($isService): ?>
-        <?= $this->icon('service', $this->translate('Service')); ?> <?= $this->qlink(
```

Comment:
Again, the icon. Why? Also here hosts got still an icon.

Reply:
see above